src/Templates/MethodTemplate.cshtml

```diff
@@ -283,13 +283,57 @@ else
     func (client @(Model.Owner)) @(Model.SenderMethodName)(req *http.Request) (@senderRetSig) {
     @if (Model.IsLongRunningOperation())
     {
+        var ftg = Model.ReturnType.Body.Cast<FutureTypeGo>();
```

Comment:
The changes here are pretty much a copy/paste of the changes removed from ModelTemplate.cshtml with some slight tweaking.